fleet/lib/rancher-backup/values.yaml

```diff
@@ -0,0 +1,18 @@
+## Default s3 bucket for storing all backup files created by the backup-restore-operator
+s3:
+  enabled: true
+  credentialSecretName: rancher-bkp
+  credentialSecretNamespace: cattle-resources-system
+  region: us-east-1
+  bucketName: rancher-bkp-${ .ClusterLabels.site }
```

Comment:
How are the buckets being configured? We probably want to make sure backups can't be deleted for period of time.  Having the config in tofu would be great but it should be at least documented somehow.

Reply:
the buckets are just regular normal bucket, can this lock be configured later?